recipe/meta.yaml

```diff
@@ -22,13 +22,13 @@ requirements:
         - python  # [win]
         - cmake  # [win]
         - msinttypes  # [win and py<35]
-        - jpeg
-        - zlib
         - bison  # [unix]
         - flex  # [unix]
+        - zlib
+        - jpeg 9b
```

Comment:
Nope. I am just following @pelson recommendation to annotate those. It serves to document that we must re-build when we update `jpeg`. Also, if conda does the pinning correctly, one day we are covered :wink: